platform-plugins.md

```diff
@@ -47,9 +47,9 @@ implementation for Android, for iOS, or for both.
 
 ### Searching for plugins
 
-Flutter plugins are shared on the [pub](https://pub.dartlang.org/) repository
-along with regular (pure-Dart) packages. To see a list of all Flutter plugins,
-use the ['Flutter Plugins'](https://pub.dartlang.org/flutter/plugins) page.
+All [available Flutter plugins](https://pub.dartlang.org/flutter/plugins) are 
```

Comment:
Is it really all? Maybe "most" ?

Or, "You can easily find Flutter plugins on pub, a package repository for Flutter and Dart."